proto/config/netcmn.proto

```diff
@@ -83,12 +83,28 @@ message ipspec {
   string subnet = 3;
   string gateway = 5;
   string domain = 6;
+  // ntp and more_ntp are used to specify several NTP servers
   string ntp = 7;
+  repeated string more_ntp = 10;
   repeated string dns = 8;
 
   // for IPAM management when dhcp is turned on.
   // If none provided, system will default pool.
   ipRange dhcpRange = 9;
+
+  // Even if DhcpType is Client, EVE still accepts static
+  // IP config and will merge it with the DHCP-provided one by default
```

Comment:
Note that the ZEDEDA controller doesn't appear to allow specifying some static info if the type is Client. I don't know what flexibility we need here (static IP, static gateway, static NTP servers, static DNS servers, static routes) when the type is client. 
Do we want to be able to specify the set of DHCP option codes to ignore? If so, can we use the DHCP option numbers instead of adding (one initially) booleans in DhcpOptionsIgnore?

Reply:
Ah - I forgot about that. Then it makes sense to have booleans. But maybe the comment for  each boolean can come with a comment saying "this means the content of DHCP option X and DHCPv6 option Y are ignored" or something to that effect.